default/scripting/specials/specials.macros

```diff
@@ -13,7 +13,7 @@ CHANCE_OF_GUARD_1
             scope = Source
             activation = AND [
                 Turn high = 0
-                Random probability = 0.60
+                Random probability = 0.70
```

Comment:
Take `GalaxyMonsteFrequency` into account here?
Something akin to: `Random probability = 0.6 + (GalaxyMonsterFrequency * 0.1)`